content/cloud/contributing.md

```diff
@@ -60,24 +63,66 @@ Create a new remote for the upstream (a pointer to the original repository to wh
 $ git remote add upstream https://github.com/fermyon/developer
 ```
 
-## 5. Add Changes
+## 5. Code Blocks and Annotations
+
+It is highly recommended that you use either the `<!-- @selectiveCpy -->` or the `<!-- @nocpy -->` annotation before each of your code blocks, and that each code block defines the appropriate [syntax highlighting](https://rdmd.readme.io/docs/code-blocks#language-support). 
+
+The selective copy annotation (`<!-- @selectiveCpy -->`) is intended for use when communicating code and/or CLI commands for the reader to copy and paste. The selective copy annotation allows the reader to see the entire code block (both commands and results) but only copies the lines that start with `$` into the readers clipboard (minus the `$`) when the user clicks the copy button. For example, copying the following code block will only copy `echo "hello"` into your clipboard, for pasting.
+
+```bash
+$ echo "hello"
+hello
+```
+
+> Note: If the command, that starts with `$`, is deliberately spread over two lines (by escaping the newline character), then the copy mechanism will still copy the second line which is technically still part of that single command.
+
+The no copy annotation (`<!-- @nocpy -->`) preceeds a code block where no copy and pasting of code is intended. For example, use no copy when just displaying output or information to demonstrate a terminal's output.
+
```

Comment:
I suggest adding the language tag of `text` to any code blocks which warrant the `<!-- @nocpy -->` as well.

Reply:
Done, thanks.